engine/src/main/java/com/datatorrent/stram/api/StramEvent.java

```diff
@@ -493,6 +514,20 @@ public void setErrorMessage(String errorMessage)
       this.errorMessage = errorMessage;
     }
 
+    public String getLogFileName()
```

Comment:
I am not convinced with moving LogFileInformation in parent class as I see the chances of adding LogFileInfo for all types of events are less and may not happen in near future. Also unless we implement it for other events ,all of them will keep writing fileName=null and offset=0 as event fields.
But considering there is no functional impact I am moving the field to parent class, and let's take this feature to completion.